zcash_client_backend/src/data_api.rs

```diff
@@ -46,6 +51,155 @@ pub enum NullifierQuery {
     All,
 }
 
+/// Balance information for a value within a single shielded pool in an account.
```

Comment:
```suggestion
/// Balance information for a value within a single shielded pool in an account.
```
Will this only ever be used for shielded pools? It seems applicable to the transparent pool as well.